### PR TITLE
[sw/silicon_creator] Use sram_ctrl DIFs in mask_rom_epmp_test.c

### PR DESCRIPTION
**Pull request by alphan**
Signed-off-by: Alphan Ulusoy <alphan@google.com>

### CODE REVIEW

File: sw/device/silicon_creator/mask_rom/mask_rom_epmp_test.c [hunk 1]
```diff
@@ -280,9 +266,12 @@ static void test_noexec_rodata(void) {
  * Test that the .data section in RAM is not executable.
  */
 static void test_noexec_rwdata(void) {
-  if (!sram_exec_enable(TOP_EARLGREY_SRAM_CTRL_MAIN_REGS_BASE_ADDR)) {
-    base_printf("failed to enable main RAM execution\n");
-  }
+  dif_sram_ctrl_t sram_ctrl;
+  CHECK(dif_sram_ctrl_init(
```
Comment (timothytrippel) [inline sw/device/silicon_creator/mask_rom/mask_rom_epmp_test.c:270]:
```suggestion
  CHECK_DIF_OK(dif_sram_ctrl_init(
```

Comment (timothytrippel) [reply sw/device/silicon_creator/mask_rom/mask_rom_epmp_test.c:270]: applies throughout

**Review by timothytrippel — APPROVED** [review]

File: sw/device/silicon_creator/mask_rom/mask_rom_epmp_test.c (hunk 1 above)
Comment (timothytrippel) [reply sw/device/silicon_creator/mask_rom/mask_rom_epmp_test.c:270]: Ah, my apologies I missed that.

Comment (alphan) [reply sw/device/silicon_creator/mask_rom/mask_rom_epmp_test.c:270]: No worries, it doesn't show up in the diff.

**Pull request merged**